lib/service_check.pm

```diff
@@ -131,6 +132,12 @@ our $default_services = {
         srv_proc_name => 'vsftpd',
         support_ver   => '12-SP2,12-SP3,12-SP4,12-SP5,15,15-SP1'
     },
+    kdump => {
+        srv_pkg_name       => 'kdump',
+        srv_proc_name      => 'kdump',
+        support_ver        => '12-SP2,12-SP3,12-SP4,12-SP5,15,15-SP1',
```

Comment:
why does it need fixed list of SLE versions? it doesn't look right as you will have to change it with each new release

Reply:
I think here is better ( and for all other `services` in this file) use blacklist than whitelist .

Reply:
we need to check this service before/after migration for these versions. all the service check function follow this kind of design.

Reply:
well, it should use same logic like other versions check are done, like  12-sp2+, <15 etc...

Reply:
As already mentioned, listing  of specific version is not optimal, but it is not in the scope of this PR.

Reply:
yes, we may create another ticket to fix this and thanks for the suggestion.